diff --git a/Data/user/Config/Dolphin.ini b/Data/user/Config/Dolphin.ini
index 63b33be4b3..945bda7667 100644
--- a/Data/user/Config/Dolphin.ini
+++ b/Data/user/Config/Dolphin.ini
@@ -1,10 +1,9 @@
 [General]
-ISOPaths = 2
+ISOPaths = 1
 RecursiveISOPaths = False
-WiiSDCardPath = ./user/Wii/sd.raw
-WiiSDCardSyncFolder = ./user/Load/WiiSDSync/
-ISOPath0 = ./Launcher
-ISOPath1 = ./Games
+WiiSDCardPath = /var/data/project-plus-dolphin/user/Load/WiiSD.raw
+WiiSDCardSyncFolder = /var/data/project-plus-dolphin/user/Load/WiiSDSync/
+ISOPath0 = /var/data/project-plus-dolphin/user/Wii/Launcher
 [NetPlay]
 SelectedHostGame = Project+ Netplay Launcher.dol
 TraversalChoice = traversal
\ No newline at end of file
